tests/spec/unit/lib/simctlHelper.spec.js

```diff
@@ -0,0 +1,502 @@
+/**
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+*/
+
+const fs = require('node:fs');
+const path = require('node:path');
+const simctl = require('simctl');
+const { events } = require('cordova-common');
+
+let json;
+let simctlHelper;
+let emitSpy;
+
+function fixtureJson (output) {
+    const file = path.resolve(__dirname, `../fixtures/${output}`);
+    return JSON.parse(fs.readFileSync(file, { encoding: 'utf-8' }).toString());
+}
+
+if (process.platform === 'darwin') {
```

Comment:
No need for these to be wrapped in a darwin check, since we're mocking the actual calls